content/en/docs/setup/minikube.md

```diff
@@ -192,48 +249,42 @@ minikube start \
     --extra-config=kubelet.image-service-endpoint=/var/run/crio.sock \
     --bootstrapper=kubeadm
 ```
-
-#### rkt container engine
-
+{{% /tab %}}
+{{% tab name="rkt container engine" %}}
 To use [rkt](https://github.com/rkt/rkt) as the container runtime run:
-
 ```shell
 minikube start \
     --network-plugin=cni \
     --enable-default-cni \
     --container-runtime=rkt
 ```
-
 This will use an alternative minikube ISO image containing both rkt, and Docker, and enable CNI networking.
+{{% /tab %}}
+{{< /tabs >}}
 
-### Driver plugins
-
-See [DRIVERS](https://git.k8s.io/minikube/docs/drivers.md) for details on supported drivers and how to install
-plugins, if required.
-
-### Use local images by re-using the Docker daemon
+#### Use local images by re-using the Docker daemon
 
-When using a single VM of Kubernetes, it's really handy to reuse the Minikube's built-in Docker daemon; as this means you don't have to build a docker registry on your host machine and push the image into it - you can just build inside the same docker daemon as minikube which speeds up local experiments. Just make sure you tag your Docker image with something other than 'latest' and use that tag while you pull the image. Otherwise, if you do not specify version of your image, it will be assumed as `:latest`, with pull image policy of `Always` correspondingly, which may eventually result in `ErrImagePull` as you may not have any versions of your Docker image out there in the default docker registry (usually DockerHub) yet.
+When using a single VM for Kubernetes, it's useful to reuse Minikube's built-in Docker daemon. This means you don't have to build a Docker registry on your host machine and push the image into it. Instead, you can build a Docker daemon inside Minikube which speeds up local experiments. Just make sure you tag your Docker image with something other than 'latest' and use that tag while you pull the image. If you do not specify the version of your image, it is assumed as `:latest` and if the image pull policy is set to  `Always`, this eventually results in an `ErrImagePull` as you may not have the Docker image in the default Docker registry (usually DockerHub). Therefore, remember to turn off the `imagePullPolicy:Always` because it would not allow Kubernetes to use the images you built locally.
```

Comment:
@shavidissa This is an improvement over the current content, but there's room for further improvement (and also some changes that introduce errors):

> When using a single VM for Kubernetes, it's useful to reuse Minikube's built-in Docker daemon. Reusing the built-in daemon means you don't have to build a Docker registry on your host machine and push the image into it. Instead, you can build inside the same Docker daemon as Minikube, which speeds up local experiments. 
>
> {{< note > }}Be sure to tag your Docker image with something other than `latest` and use that tag to pull the image. Because `:latest` is the default value, with a corresponding default image pull policy of `Always`, an image pull error (`ErrImagePull`) eventually results if you do not have the Docker image in the default Docker registry (usually DockerHub).{{< /note >}}

Reply:
Nice! Thanks a lot.. the messaging is much more clear here.